spec/fortnox/api_spec.rb

```diff
@@ -5,10 +5,14 @@
 
   context 'get access token' do
     before do
-      ENV['FORTNOX_API_BASE_URL'] = 'http://api.fortnox.se/3/'
-      ENV['FORTNOX_API_CLIENT_SECRET'] = 'P5K5vE3Kun'
-      ENV['FORTNOX_API_ACCESS_TOKEN'] = '3f08d038-f380-4893-94a0-a08f6e60e67a'
-      ENV['FORTNOX_API_AUTHORIZATION_CODE'] = 'ea3862b0-189c-464b-8e23-1b9702365ea1'
+      stub_const(
```

Comment:
A lot of these now ... How about declaring a helper that does the around stuff:

```
def stub_environment( hash )
  stub_const( 'ENV', ENV.to_hash.merge( hash ))
end
```
and just call it like this:
```
stub_environment(
  'FORTNOX_API_BASE_URL' => 'http://api.fortnox.se/3/',
  'FORTNOX_API_CLIENT_SECRET' => 'P5K5vE3Kun',
  'FORTNOX_API_ACCESS_TOKEN' => '3f08d038-f380-4893-94a0-a08f6e60e67a',
  'FORTNOX_API_AUTHORIZATION_CODE' => 'ea3862b0-189c-464b-8e23-1b9702365ea1'
)
```